tools/convert_pkl_to_pb.py

```diff
@@ -124,10 +125,41 @@ def unscope_name(name):
 
 
 def reset_names(names):
-    for i in range(0, len(names)):
+    for i in range(len(names)):
         names[i] = unscope_name(names[i])
 
 
+def convert_collect_and_distribute(
+    op, blobs,
+    roi_canonical_scale,
+    roi_canonical_level,
+    roi_max_level,
+    roi_min_level,
+    rpn_max_level,
+    rpn_min_level,
+    rpn_post_nms_topN,
+):
+    print('Converting CollectAndDistributeFpnRpnProposals'
+          ' Python -> C++:\n{}'.format(op))
+    assert op.name.startswith('CollectAndDistributeFpnRpnProposalsOp'), \
+        'Not valid CollectAndDistributeFpnRpnProposalsOp'
+
+    inputs = [x for x in op.input]
+    ret = core.CreateOperator(
+        'CollectAndDistributeFpnRpnProposals',
+        inputs,
```

Comment:
Hi, Iam trying to convert maskrcnn model from pkl to pb format. But Iam getting below error
Traceback (most recent call last):
  File "convert_pkl_to_pb.py", line 586, in <module>
    main()
  File "convert_pkl_to_pb.py", line 574, in main
    [net, init_net] = convert_model_gpu(args, net, init_net)
  File "convert_pkl_to_pb.py", line 300, in convert_model_gpu
    ret = core.InjectDeviceCopiesAmongNets([ret_init_net, ret_net])
  File "/usr/local/anaconda3/envs/caffetwo/lib/python2.7/site-packages/caffe2/python/core.py", line 2407, in InjectDeviceCopiesAmongNets
    blob_remap=blob_remap,
  File "/usr/local/anaconda3/envs/caffetwo/lib/python2.7/site-packages/caffe2/python/core.py", line 2298, in InjectCrossDeviceCopies
    "input {} should be defined in the net.".format(input)
AssertionError: input rpn_rois should be defined in the net.

Input to the script is config file = e2e_mask_rcnn_R-101-FPN_1x.yaml and its pkl model

Reply:
> AssertionError: input rpn_rois should be defined in the net.
Looks like you also need #449 PR to do what you want.

Reply:
> AssertionError: input rpn_rois should be defined in the net.
Looks like you also need #449 PR to do what you want.

Reply:
> AssertionError: input rpn_rois should be defined in the net.

Looks like you also need #449 PR to do what you want.

Reply:
Hi, Iam able to convert the model from pkl to pb format with convert_pkl_to_pb.py with args.device = cpu. But when Iam viewing the network using netron (https://github.com/lutzroeder/Netron), mask_conv_body section is missing in the graph. I want to understand if this is the issue with convert_pkl_to_pb.py or with Netron.